src/queries/role.ts

```diff
@@ -14,7 +15,9 @@ import type {
 
 type ByRoleOptions = {
   name?: TextMatch;
-};
+} & AccessibilityState;
+
+type AccessibilityStateField = keyof AccessibilityState;
```

Comment:
nit: more typical name would be using Key suffix
```suggestion
type AccessibilityStateKey = keyof AccessibilityState;
```